### ISSUE TITLE
What are Nature's particle accelerators?

### ISSUE THREAD

**Issue by cwjames1983** [ethrane/transients#7]
Only a fraction of the high-energy gamma-rays, cosmic rays, and neutrinos have origins which can be associated with astronomical sources. One two Galactic accelerators up to PeV energies, and one extragalactic source, have been confirmed identified. A large fraction of all TeV gamma-ray sources are yet to be identified (Sabrina: fraction?);  no clear correlation between either high-energy neutrinos and ultra-high-energy cosmic rays and any given source class is yet to be found. This is important for understanding the physics supernovae, AGN and jet physics, microquasars, young stellar associations, and more.

**Comment by cwjames1983** [comment]
Forgot to add GRBs to this

**Comment by adellej** [comment]
TDEs also seem to be efficient particle accelerators, and can produce relativistic jets

**Comment by kauchettl** [comment]
I agree that this is an important question @cwjames1983 and has interesting connections to a lot of the phenomena that we as a community are studying. However, I think rather than it being a separate question which has the risk of being too focused such that it is not necessarily selected by the Decadal as one of the main questions that is driving the science we are doing, it could be nicely incorporated into a broader question. This could either be an iteration of "how does the night sky/universe vary on human timescales?" or if we happen to expand "how do stars die?" (or even the other potential questions that have been proposed). Similar to how one would discuss how X-ray, optical/IR and radio would help us probe the nature of these events, we would then naturally highlight how high-energy gamma-rays, cosmic rays, and neutrinos also play a key part in our science and for many in the community, so that that we are covering the full multi-wavelength and multi-messenger spectrum.

**Comment by gemmaanderson** [comment]
GRBs are a really important science case for CTA support. Only about 6 GRBs have been detected at TeV energies, with the first in 2019 and these discoveries have revitalised the field. We still don't understand what the emission mechanism is as the presumed synchrotron self-comptonisation/inverse Compton scattering often don't describe the spectra.

Short GRBs are also gravitational wave events so there is lots of multi-messenger crossover with EM, high energy gamma-rays and gravitational waves.

**Comment by GavinRowell** [comment]
Yes, CTA should probe GRBs out to z~4 or so (based on some historical examples) and will be able to look for sub-minute variations in energy spectra (something well-beyond Fermi-LAT's capability).

**Comment by GavinRowell** [comment]
LHAASO has now detected over a dozen PeV gamma-ray sources in the MW so we know multi-PeV particle acceleration occurs in quite a few places. Still lots of debate about their origin but old pulsar wind nebulae, stellar clusters or even some SNRs are being discussed (besides the obvious PeV acceleration occurring in the Galactic Centre).

**Comment by GavinRowell** [comment]
This question applies to both cosmic rays (protons, nuclei) and electrons since the fundamental acceleration processes still have many unsolved aspects (how are electrons injected into shocks, what is the role of magnetic reconnection vs. diffusive shock acceleration, what role to jets play etc etc.....). Transient observations from radio to gamma rays plus neutrinos on short timescales (<days, hrs...) are needed to take the next steps.
